backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/vdscommands/StorageServerConnectionManagementVDSParameters.java

```diff
@@ -10,6 +10,7 @@
 public class StorageServerConnectionManagementVDSParameters extends GetStorageConnectionsListVDSCommandParameters {
     private StorageType privateStorageType;
     private boolean sendNetworkEventOnFailure = true;
+    private List<String> privateDeviceList = null;
```

Comment:
no need to initialize to null, and I'd rename to just `deviceList`, I'm not sure what private means